emails/tests/views_tests.py

```diff
@@ -496,7 +498,8 @@ def test_spamVerdict_FAIL_auto_block_doesnt_relay(self) -> None:
         assert self.ra.num_forwarded == 0
 
     def test_domain_recipient(self) -> None:
-        _sns_notification(EMAIL_SNS_BODIES["domain_recipient"])
+        with self.assertLogs("glean", "INFO") as caplog:
```

Comment:
praise: TIL `assertLogs` and `assertNoLogs`. question (non-blocking): when do we use `assertLogs` vs. a pytest fixture?

Reply:
`caplog` when the test is a function, `assertLogs` when it is a method in a `unittest.TestCase` class